test-case/check-userspace-cardinfo.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+##
+## Case Name: check-userspace-cardinfo
+## Preconditions:
+##    N/A
+## Description:
+##    Get the card name from pactl info
+##    check if there is an available card to do the following tests
+## Case step:
+##    1. run pactl to get the cards info
+##    2. check the card name
+## Expect result:
+##    There is at least one available card for test
+##
+
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+# check pulseaudio runs properly or not
+if ! func_lib_check_pa; then
+    die "Please check whether pulseaudio runs correctly or not"
+fi
+
+func_opt_parse_option "$@"
+
+OLDIFS=$IFS
+dlogc "pactl list cards short"
+cardlist=$(pactl list cards short)
+((available_card=0))
```

Comment:
This bashism not compatible with `set -e`. Change to the POSIX standard
```shell
: $((available_card=0))
```

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_04